src/presets.asm

```diff
@@ -273,6 +273,9 @@ preset_start_gameplay:
     JSL $878016  ; Clear animated tile objects
     JSL $88829E  ; Wait until the end of a v-blank and clear (H)DMA enable flags
 
+    ; Set Samus last pose same as current pose
+    LDA !SAMUS_POSE : STA $0A20
+
```

Comment:
This causes a problem for presets that were saved while sparking. Before this change, Samus would start moving slowly and build up speed until crashing the spark normally. Now, it just traps Samus in the air with the sparking pose. I'm not sure how to handle it yet. Could get a bit messy to check for each spark pose, but I guess that's the last resort.

Unrelated to that, I just noticed that custom presets crash if saved during the spark windup pose.